src/util.ts

```diff
@@ -305,6 +305,44 @@ export async function mouseup(client: Client, selector: string, scale: number) {
     })
 }
 
+export async function clear(client: Client, selector: string): Promise<void> {
+    await wait(500)
+    await focus(client, selector)
+
+    const {Input} = client
+
+    const text = await getValue(client, selector)
+
+    const optionsDelete = {
+        nativeVirtualKeyCode: 46,
+        windowsVirtualKeyCode: 46,
+    }
+
+    const optionsBackspace = {
+        nativeVirtualKeyCode: 8,
+        windowsVirtualKeyCode: 8,
+    }
+
+    for (let i = 0; i < text.length; i++) {
+        await Input.dispatchKeyEvent({
+            ...optionsDelete,
+            type: 'rawKeyDown',
+        })
+        Input.dispatchKeyEvent({
```

Comment:
without wait for second resolution, it executes pretty much faster and it acts as more natural clear. However this could be changed with `await press(client, 8, text.length); await press(client, 46, text.length);`